src/EnergyPlus/CMakeLists.txt

```diff
@@ -880,8 +879,27 @@ install(TARGETS energyplus energyplusapi DESTINATION ./)
 
 if(LINK_WITH_PYTHON)
   # link the executable to the dll and bring it in
-  get_filename_component(RESOLVED_PYTHON_LIBRARY "${PYTHON_LIBRARIES}" REALPATH)
+  get_filename_component(RESOLVED_PYTHON_LIBRARY "${Python_LIBRARIES}" REALPATH)
   if(WIN32)
+
+    # In case you have both release and debug Python libraries on your system, Python_LIBRARIES might be "optimized;C:/.../python38.lib;debug;C:/.../python38_d.lib"
+    # so it can't be used directly, we could use a generator expression to find the $<TARGET_FILE:Python::Python> library used and that'd point directly
+    # to the DLL. But it won't work nicely for the install(code ...) command below... so we do a hack, get the optimized one always...
+    list(LENGTH Python_LIBRARIES _LEN)
+    if (_LEN GREATER 1)
+      set (_Python_DOING_RELEASE FALSE)
+      foreach(_currentArg ${Python_LIBRARIES})
+        if ("x${_currentArg}" STREQUAL "xoptimized")
+          set(_Python_DOING_RELEASE TRUE)
+        elseif(_Python_DOING_RELEASE)
+          get_filename_component(RESOLVED_PYTHON_LIBRARY "${_currentArg}" REALPATH)
+          break()
+        endif()
```

Comment:
Is there a code path here missing in case you don't find the optimized keyword?  I know it would be random but just trying to make it as robust as possible.  Maybe if the finder finds multiple debug build versions on one system??  Would it be `debug;C:/Something.lib;debug;C:/SomethingElse.lib`?

Reply:
@Myoldmopar  AFAIK you cannot install only debug libraries, and optimized are always installed. Either you have both and it's guaranteed you'll have optimized, or you have only one item. See source code here: https://gitlab.kitware.com/cmake/cmake/blob/fc762735196dbbc983dfac2145cd7afd7892bf54/Modules/FindPython/Support.cmake#L63-91

Reply:
OK thanks.